allennlp/models/simple_tagger.py

```diff
@@ -141,6 +136,16 @@ def tag(self, text_field: TextField) -> Dict[str, Any]:
         predictions = output_dict["class_probabilities"].data.squeeze(0)
         _, argmax = predictions.max(-1)
         indices = argmax.squeeze(1).numpy()
-        tags = [self.vocabulary.get_token_from_index(x, namespace="tags") for x in indices]
+        tags = [self.vocab.get_token_from_index(x, namespace="tags") for x in indices]
 
         return {"tags": tags, "class_probabilities": predictions.numpy()}
+
+    @classmethod
+    def from_params(cls, vocab: Vocabulary, params: Params) -> 'SimpleTagger':
```

Comment:
Doesn't having non-param arguments here slightly defeat the point of this method? I see why, because the vocab is instantiated before, but still...